python/ray/tune/function_runner.py

```diff
@@ -59,6 +61,7 @@ def _get_and_clear_status(self):
         with self._lock:
             res = self._latest_result
             self._latest_result = None
+            res["training_iteration"] = self._iteration
```

Comment:
```suggestion
            res.setdefault(TRAINING_ITERATION, self._iteration)
```

Reply:
(and import as necessary)

Reply:
Also can you do the same for the above "if self._done and not self._latest_result:" case?